crypto/src/hash.rs

```diff
@@ -306,6 +306,44 @@ pub fn chain_id_from_block_hash(block_hash: &BlockHash) -> Result<ChainId, Blake
         .unwrap_or_else(|_| unreachable!("ChainId is created from slice of correct size")))
 }
 
+#[derive(Debug, Error)]
+pub enum TryFromPKError {
```

Comment:
@akoptelov 
isnt it possible to refactor/use/clean a `ConversionError`  here instead of new `TryFromPKError` ?

Reply:
No, it's not. `ConversionError` is defined for signatures, in `tezos_messages` crate, and is quite overloaded to be used here. Here is the error that might happen solely for getting a hash of one typed hash and storing it into another.